misk-core/src/main/kotlin/misk/sampling/RateLimiter.kt

```diff
@@ -107,6 +108,35 @@ class RateLimiter private constructor(
     }
   }
 
+  /**
+   * Returns the maximum number of permits that could have
+   * been acquired by a call to [tryAcquire], assuming the caller
+   * passed the same [timeout] and [unit].
+   *
+   * @return the permits remaining.
+   */
+  fun getPermitsRemaining(
+    unit: TimeUnit,
+    timeout: Long
+  ): Long {
+    val allocatedUntil = atomicAllocatedUntil.get()
+    println("getPermitsRemaining.allocatedUntil: $allocatedUntil")
```

Comment:
this should be removed before merge

Reply:
(I'm assuming it is there solely for debugging during development and not intended to land in master)